### DISCUSSION TITLE
collective badges or badges in general

### DISCUSSION THREAD

**Discussion by Baggiest** [benawad/dogehouse#1943] [Ideas]
can we do a badge system kinda like discord, just for like giving out badges if there is a really special event, just to like motivate the community to do these kinda activities 

or maybe a dev badge for the people who are actively helping to build the platform or even if you guys ever tried to monetize the app by a subscription-based(or donation) model u can have a backers badge

sorry if it sounds dumb but it's a thing I thought I need to share with you guys

**Comment by amitojsingh366** [comment]
these are being implemented, just recently the "Bot" badge was added

**Comment by amitojsingh366** [comment]
two new badges, the "DogeHouse Contributor" and "DogeHouse Staff" badges have been added in #2654

**Reply by Baggiest** [reply]
Lets go
